ci-operator/config/ComplianceAsCode/content/ComplianceAsCode-content-master__4.17.yaml

```diff
@@ -1,12 +1,20 @@
 build_root:
   image_stream_tag:
-    name: release
```

Comment:
This image contained `jq`:

```
     helpers.go:1237: Failed applying remediation '/go/src/github.com/ComplianceAsCode/content/applications/openshift/api-server/audit_log_forwarding_enabled/tests/ocp4/e2e-remediation.sh': exit status 127
        ++ oc version -ojson
        ++ jq -r .openshiftVersion
        /go/src/github.com/ComplianceAsCode/content/applications/openshift/api-server/audit_log_forwarding_enabled/tests/ocp4/e2e-remediation.sh: line 4: jq: command not found
        + ocp_version= 
```

The new image we're using below doesn't - so we'll need to add it somewhere.

Reply:
Should be resolved with `rhel-9-golang-1.22-ci-build-root-multi-openshift-4.18`